cli/command/container/opts.go

```diff
@@ -210,7 +210,7 @@ func addFlags(flags *pflag.FlagSet) *containerOptions {
 	flags.SetAnnotation("cgroupns", "version", []string{"1.41"})
 
 	// Network and port publishing flag
-	flags.Var(&copts.extraHosts, "add-host", "Add a custom host-to-IP mapping (host:ip)")
+	flags.Var(&copts.extraHosts, "add-host", "Add a custom host-to-IP mapping (host:ip, or host=ip)")
```

Comment:
I'm slightly leaning towards "soft deprecating" the old format, and not advertising it too prominent.

In the docs, we could still make a mention of the old format (for users who are running an older version of the cli)

Reply:
Ah, yes - good idea! Will do, thank you.